src/main/java/uk/gov/register/presentation/dao/Entry.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.register.presentation.dao;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.sql.Timestamp;
```

Comment:
I agree we must not use Timestamp class for tis purpose. java8 got new DateTime API which is immutable and much improved version, can use that.